tests/gpflow/optimizers/test_scipy.py

```diff
@@ -63,4 +63,7 @@ def get_values(model):
     # The tolerance of the following test had to be loosened slightly from atol=1e-15
     # due to the changes introduced by PR #1213, which removed some implicit casts
     # to float32. With TF 2.4 / TFP 0.12, we had to further increase atol from 1e-14.
+    np.testing.assert_allclose(
+        get_values(m1), get_values(m2), rtol=4e-14, atol=2e-14
+    )  # for Predictor
```

Comment:
What do you mean by "for Predictor"?

Reply:
removed line - not actually needed

Reply:
(this was left over from tests where SVGP.predict_f wasn't using the fused_predict_f code path but the precomputed one, which slightly deteriorated performance; probably a merge remnant)